### PR TITLE
Updated Readme file to include definition of the datatable library

### PR DESCRIPTION
**Pull request by parulnith**
Updated Readme file to include a definition of the datatable library. This would help the readers to understand the purpose of the library and how this can provide better alternatives than some of the existing ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,9 @@
 ## Mission 🚀
 To provide 100 datatable exercises over different sections structured as a course or tutorials to teach and learn for beginners, intermediates as well as experts.
 
-Python's [datatable](https://github.com/h2oai/datatable) package that is synonymous with R's [data.table](https://github.com/Rdatatable/data.table) library can be used for various kinds of data processing, feature engineering and modelling tasks as part of data science pipelines.
+## Datatable
```
Comment (vopani) [inline README.md:17]: Can you suggest an emoji that can be used here?

File: README.md [hunk 2]
```diff
@@ -14,7 +14,9 @@
 ## Mission 🚀
 To provide 100 datatable exercises over different sections structured as a course or tutorials to teach and learn for beginners, intermediates as well as experts.
 
-Python's [datatable](https://github.com/h2oai/datatable) package that is synonymous with R's [data.table](https://github.com/Rdatatable/data.table) library can be used for various kinds of data processing, feature engineering and modelling tasks as part of data science pipelines.
+## Datatable 
+
+Python's [datatable](https://github.com/h2oai/datatable) package is a library for efficient data processing. It is synonymous with R's [data.table](https://github.com/Rdatatable/data.table) library and can be used for various kinds of data processing, feature engineering and modeling tasks as part of data science pipelines. It specifically shines when handling large dataframes and managing  out-of-memory datasets. This toolkit resembles [pandas](https://pandas.pydata.org/) very closely but is more focused on speed and multi-threaded data processing operations.
```
Comment (vopani) [inline README.md]: Double-space after `managing`

**Review by vopani — CHANGES_REQUESTED** [review]
Thanks! Looks useful. Some minor edits required.

File: README.md (hunk 1 above)
Comment (parulnith) [reply README.md:17]: I initially thought of adding the official logo in front of the name, as follows:
Screenshot 2021-07-05 at 12 56 37 PM

The link to the logo is: https://datatable.readthedocs.io/en/latest/_static/datatable_small_logo.svg

File: README.md (hunk 2 above)
Comment (parulnith) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (vopani) [reply README.md:17]: Cool, lets see.

Comment (vopani) [reply README.md:17]: Done. Added.

**Pull request merged**